internal/manifest/manifest.go

```diff
@@ -9,6 +9,15 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/rpmmd"
 )
 
+type Arch uint64
+
+const (
+	ARCH_X86_64 Arch = iota
+	ARCH_AARCH64
+	ARCH_S390X
+	ARCH_PPC64LE
+)
```

Comment:
We already have something like this defined in https://github.com/osbuild/osbuild-composer/blob/2c2d4812d4b381275e8c52d86ec81b7949b42cfe/internal/distro/distro.go#L15-L22

Reply:
so far i have the distro package depend on all the new things, but not the other way around. let's see if we should change that once it all settles (or if we should drop things from the distro package)

Reply:
Sure, one way or the other sounds good. I just wanted to point out that we already have something similar defined, so that we don't end up with both definitions in the end.